pom.xml

```diff
@@ -135,6 +152,8 @@
                 <configuration>
                     <source>1.8</source>
                     <target>1.8</target>
+                    <testSource>11</testSource>
+                    <testTarget>11</testTarget>
```

Comment:
new versions of mockito require java 11+ to run. This allows us to have a contemporary test suite while still compiling production classes for java 8.